apps/emqx_management/src/emqx_mgmt_data_backup.erl

```diff
@@ -626,6 +626,7 @@ do_export_extra_data() -> [].
 
 -ifdef(EMQX_ENTERPRISE).
 import(Filename, OverridesJson) ->
+    _ = legal_filename(Filename) orelse error(bad_filename),
```

Comment:
We'd better use throw to avoid too large stacktrace being printed. And also add the filename itself into the error:
 `throw({bad_filename, Filename})`